components/webpack/webpack.config.js

```diff
@@ -5,9 +5,6 @@
 
 const path = require('path')
 const GenerateDepfilePlugin = require('./webpack-plugin-depfile')
-const createStyledComponentsTransformer = require('typescript-plugin-styled-components').default
```

Comment:
Can you split this in to a separate commit and have the message make it clear why it's being changed?

Reply:
Done.